example-document.tex

```diff
@@ -44,4 +44,53 @@
 % Index
 \printindex
 
+% Learning objectives
```

Comment:
Do you know if this section will be necessary after finishing all tasks within PR #90? 
This seems quite extensive and could be scary for the users without programming backround.

Reply:
The section was only meant for debugging, it must have slipped in with the other changes. I fixed this in commit 727ed35 from this PR.